src/bin/nydusd/blob_cache.rs

```diff
@@ -101,6 +101,42 @@ struct BlobCacheState {
     id_to_config_map: HashMap<String, BlobCacheObjectConfig>,
 }
 
+impl BlobCacheState {
+    fn new() -> Self {
+        Self {
+            id_to_config_map: HashMap::new(),
+        }
+    }
+
+    fn remove(&mut self, domain_id: &str) {
+        let scoped_blob_prefix = format!("{}-", domain_id);
```

Comment:
The algorithm to generate the blob key is
```
        let scoped_blob_id = if domain_id.is_empty() {
            blob_info.blob_id().to_string()
        } else {
            domain_id + "-" + blob_info.blob_id()
        };
```
So we should keep in consistence and the `domain_id` may be empty.

Reply:
The `domain_id` may be used as tenant_id when support multi-tenants.
Or the `domain_id` may be empty when support single tenant.

Reply:
for now, for bootstrap, domain_id should equal to blob_id.
I think we could live with it for now, and then think further together with kernel approach evolves...